framed.js

```diff
@@ -1,17 +1,15 @@
 'use strict';
 
-var Client = require('ifrau/client/slim'),
-	Promise = require('./promise-or-lie');
+var Client = require('./ifrau-client');
 
 var REQUEST_KEY = require('./request-key');
 
 module.exports = function getFramedJwt(scope) {
-	return Promise
-		.resolve()
-		.then(function() {
-			return new Client().connect();
-		})
+	return Client
```

Comment:
This should be functionally equivalent to what existed before. Before we had:
`Resolve empty promise -> resolve client -> resolve connection -> resolve request`
... and now we have:
`Resolve client -> resolve connection -> resolve request`

Reply:
It probably makes sense to rename these to make it explicit that the `require` is returning a promise. I'll do so.